fhir-validation/src/test/java/com/ibm/fhir/validation/test/ProfileConstraintTest.java

```diff
@@ -51,10 +51,10 @@ public void testProfileConstraint() throws Exception {
         Coverage coverage = Coverage.builder()
                 .status(CoverageStatus.ACTIVE)
                 .beneficiary(Reference.builder()
-                    .reference(string("none"))
+                    .display(string("none"))
                     .build())
                 .payor(Reference.builder()
-                    .reference(string("none"))
+                    .display(string("none"))
```

Comment:
Shouldn't this become three errors instead?

Reply:
This change doesn't introduce any additional constraint errors, so I would expect the errors to stay at 1.